src/context/online.rs

```diff
@@ -44,10 +44,15 @@ pub struct AudioContextOptions {
     /// Identify the type of playback, which affects
     /// tradeoffs between audio output latency and power consumption
     pub latency_hint: AudioContextLatencyCategory,
-    /// Sample rate of the audio Context and audio output hardware
+
+    /// Sample rate of the audio context and audio output hardware
     pub sample_rate: Option<f32>,
-    /// The audio output device - use `None` for the default device
-    pub sink_id: Option<String>,
+
+    /// The audio output device
```

Comment:
This API is terrible and I know it. The spec is not much better. I will explore other options